packages/config-types/src/index.ts

```diff
@@ -0,0 +1,952 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface ExpoConfig {
+  /**
+   * The name of your app as it appears both within Expo client and on your home screen as a standalone app.
+   */
+  name: string;
+  /**
+   * A short description of what your app is and why it is great.
+   */
+  description?: string;
+  /**
+   * The friendly URL name for publishing. For example, `myAppName` will refer to the `expo.io/@project-owner/myAppName` project.
+   */
+  slug: string;
+  /**
+   * The Expo account name of the team owner, only applicable if you are enrolled in Expo Developer Services. If not provided, defaults to the username of the current user.
+   */
+  owner?: string;
+  /**
+   * Defaults to `unlisted`. `unlisted` hides the project from search results. `hidden` restricts access to the project page to only the owner and other users that have been granted access. Valid values: `public`, `unlisted`, `hidden`.
+   */
+  privacy?: 'public' | 'unlisted' | 'hidden';
+  /**
+   * The Expo sdkVersion to run the project on. This should line up with the version specified in your package.json.
+   */
+  sdkVersion?: string;
+  /**
+   * **Note: Don't use this property unless you are sure what you're doing**
+   *
+   * The runtime version associated with this manifest for bare workflow projects. If provided, this must match the version set in Expo.plist or AndroidManifest.xml.
+   */
+  runtimeVersion?: string;
+  /**
+   * Your app version. In addition to this field, you'll also use `ios.buildNumber` and `android.versionCode` — read more about how to version your app [here](../../distribution/app-stores/#versioning-your-app). On iOS this corresponds to `CFBundleShortVersionString`, and on Android, this corresponds to `versionName`. The required format can be found [here](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring).
+   */
+  version?: string;
+  /**
+   * Platforms that your project explicitly supports. If not specified, it defaults to `["ios", "android"]`.
+   */
+  platforms?: ('android' | 'ios' | 'web')[];
+  /**
+   * If you would like to share the source code of your app on Github, enter the URL for the repository here and it will be linked to from your Expo project page.
+   */
+  githubUrl?: string;
+  /**
+   * Locks your app to a specific orientation with portrait or landscape. Defaults to no lock. Valid values: `default`, `portrait`, `landscape`
+   */
+  orientation?: ('default' | 'portrait' | 'landscape') | null;
```

Comment:
this is basically an any

Reply:
So it can be one of `'default' | 'portrait' | 'landscape', or `undefined`, or `null` 🤔